requirements.txt

```diff
@@ -1,2 +1,4 @@
+-e git+https://github.com/datasciencebr/serenata-toolbox.git@6126436#egg=serenata-toolbox
```

Comment:
@cuducos You [recently removed](https://github.com/datasciencebr/rosie/commit/9414eb3195ea7c4b9c8d9b8d073f3bbd078482e2) the `-e` from Rosie. Was for not saving it inside the project folder, right?

If that's the case, can you change for the version after @cuducos' commit?